[2024-06-19T10:30:01.109+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T10:30:01.129+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:50:00+00:00 [queued]>
[2024-06-19T10:30:01.139+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:50:00+00:00 [queued]>
[2024-06-19T10:30:01.139+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T10:30:01.162+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 04:50:00+00:00
[2024-06-19T10:30:01.164+0530] {standard_task_runner.py:63} INFO - Started process 4771 to run task
[2024-06-19T10:30:01.167+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'scheduled__2024-06-19T04:50:00+00:00', '--job-id', '132', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmpsx4rnqlh']
[2024-06-19T10:30:01.169+0530] {standard_task_runner.py:91} INFO - Job 132: Subtask backup_task
[2024-06-19T10:30:01.223+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:50:00+00:00 [running]> on host FN-21052948.
[2024-06-19T10:30:01.290+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T04:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-19T04:50:00+00:00'
[2024-06-19T10:30:01.291+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T10:30:01.299+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T10:30:01.299+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-19T10:30:01.303+0530] {subprocess.py:86} INFO - Output:
[2024-06-19T10:30:01.310+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:30:01.315+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:30:01.326+0530] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:30:01.510+0530] {subprocess.py:93} INFO - Backup completed in 0 seconds
[2024-06-19T10:30:01.511+0530] {subprocess.py:93} INFO - Backup Success in location: /home/kiit/airflow/backup | Required Space: 0GB, Available Space: 245.31GB
[2024-06-19T10:30:01.511+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T10:30:01.511+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T10:30:01.534+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=scheduled__2024-06-19T04:50:00+00:00, execution_date=20240619T045000, start_date=20240619T050001, end_date=20240619T050001
[2024-06-19T10:30:01.581+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T10:30:01.598+0530] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T10:30:01.606+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-19T10:30:28.548+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-19T10:30:28.565+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:50:00+00:00 [queued]>
[2024-06-19T10:30:28.570+0530] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:50:00+00:00 [queued]>
[2024-06-19T10:30:28.571+0530] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-19T10:30:28.588+0530] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): backup_task> on 2024-06-19 04:50:00+00:00
[2024-06-19T10:30:28.592+0530] {standard_task_runner.py:63} INFO - Started process 4809 to run task
[2024-06-19T10:30:28.595+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Backup_Database', 'backup_task', 'scheduled__2024-06-19T04:50:00+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/basic.py', '--cfg-path', '/tmp/tmpuy0yeqfc']
[2024-06-19T10:30:28.596+0530] {standard_task_runner.py:91} INFO - Job 133: Subtask backup_task
[2024-06-19T10:30:28.640+0530] {task_command.py:426} INFO - Running <TaskInstance: Backup_Database.backup_task scheduled__2024-06-19T04:50:00+00:00 [running]> on host FN-21052948.
[2024-06-19T10:30:28.706+0530] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarika' AIRFLOW_CTX_DAG_ID='Backup_Database' AIRFLOW_CTX_TASK_ID='backup_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-19T04:50:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-19T04:50:00+00:00'
[2024-06-19T10:30:28.707+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-19T10:30:28.713+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-06-19T10:30:28.714+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/kiit/airflow/bashScripts/backup.sh ']
[2024-06-19T10:30:28.718+0530] {subprocess.py:86} INFO - Output:
[2024-06-19T10:30:28.723+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:30:28.727+0530] {subprocess.py:93} INFO - mysql: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:30:28.738+0530] {subprocess.py:93} INFO - mysqldump: [Warning] Using a password on the command line interface can be insecure.
[2024-06-19T10:30:28.923+0530] {subprocess.py:93} INFO - Backup completed in 0 seconds
[2024-06-19T10:30:28.923+0530] {subprocess.py:93} INFO - Backup Success in location: /home/kiit/airflow/backup | Required Space: 0GB, Available Space: 245.31GB
[2024-06-19T10:30:28.923+0530] {subprocess.py:97} INFO - Command exited with return code 0
[2024-06-19T10:30:28.923+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-19T10:30:28.950+0530] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Backup_Database, task_id=backup_task, run_id=scheduled__2024-06-19T04:50:00+00:00, execution_date=20240619T045000, start_date=20240619T050028, end_date=20240619T050028
[2024-06-19T10:30:28.969+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-19T10:30:28.985+0530] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-19T10:30:28.991+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
